collect_app/src/test/java/org/odk/collect/android/database/DatabaseFormsRepositoryTest.java

```diff
@@ -28,6 +30,18 @@ public FormsRepository buildSubject() {
         return new DatabaseFormsRepository();
     }
 
+    @Override
+    public FormsRepository buildSubject(Clock clock) {
+        RobolectricHelpers.overrideAppDependencyModule(new AppDependencyModule() {
```

Comment:
This could also go in an `@Before` but I do think I prefer it here. It's nice that we end up with `Clock` being passed into the `buildSubject` so it's super clear to the implementers they should build their version wih it.